pom.xml

```diff
@@ -60,10 +60,10 @@
     </scm>
 
     <properties>
-        <spring.version>5.2.13.RELEASE</spring.version>
+        <spring.version>5.2.22.RELEASE</spring.version>
```

Comment:
Are you sure the 5.2.22.RELEASE doesn't have spring-beans critical CVE?

Reply:
Bumped to 5.3.23.RELEASE